tgui/packages/tgui/interfaces/ChemiCompiler/ChemiCompilerTextArea.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Garash2k (https://github.com/Garash2k)
+ * @license ISC
+ */
+
+import { useBackend } from '../../backend';
+import { ChemiCompilerData } from './type';
+import { TextArea } from '../../components';
+
+export const ChemiCompilerTextArea = (_props, context) => {
+  const { act, data } = useBackend<ChemiCompilerData>(context);
+  const { inputValue, loadTimestamp, theme } = data;
+  return (
+    <TextArea
+      value={inputValue}
+      onInput={(_event, value) => { act('updateInputValue', { value }); }}
+      height="100%"
+      // The load button would break if we pressed it between the input's act and the next refresh.
+      // This ensures a refresh after every load button click
+      key={loadTimestamp}
+      fontFamily="Consolas"
```

Comment:
Please include a backup font of `monospace`. I've included what I think it could be below but not as a suggestion because I haven't verified syntax/that this works. Set it to something like `"thisfontdoesnotexist, monospace"` and check behavior.

```
      fontFamily="Consolas, monospace"
```

Future work: would be good to augment `TextArea`'s this way so you can just pass in a `monospace` prop.

Reply:
Does work: ![alt text](image.png)